components/setup/BuildInstruction.tsx

```diff
@@ -0,0 +1,110 @@
+import axios from 'axios';
+import type { Session } from 'next-auth';
+import React, { useEffect, useState } from 'react';
+import { RepoProvider } from '../../utils/providerAPI';
+import { CloudBuildStatus } from '../../utils/pubsub/pubsubClient';
+import Button from '../Button';
+import DockerInstructions from './DockerInstructions';
+import RepoSelection from './RepoSelection';
+
+interface BuildInstructionProps {
+	selectedHosting: string;
+	selectedProvider: RepoProvider;
+	selectedInstallationType: string;
+	userId: string;
+	session: Session | null;
+}
+
+const BuildStatus: React.FC<{ buildStatus: CloudBuildStatus | null, isButtonDisabled: boolean }> = ({ buildStatus, isButtonDisabled }) => {
```

Comment:
LOW LOGAF: but can we have better naming for `isButtonDisabled` state? I don't understand which button are we talking about?

Reply:
Hmm... this was already there; so I didn't touch it. But yes, we can rename it to something like `isTriggerBuildButtonDisabled` to make it more descriptive.

Waise there is only one button in this component.

Reply:
Done this.